### PR TITLE
update heisenbug documentation page

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
I had the recompile seissol recently on heisenbug, and I'm taking the opportunity to update the documentation.

### CODE REVIEW

File: Documentation/heisenbug.rst [hunk 1]
```diff
@@ -10,67 +10,84 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
...
-.. code-block:: bash
-
-    -DDEVICE_ARCH=sm_86 -DHOST_ARCH=hsw -DDEVICE_BACKEND=cuda -DPRECISION=single -DHIPSYCL_CUDA_PATH=$CUDA_HOME
+    spack compiler find
+    # deactivate conda module (if any)
+    conda deactivate
+    # python has an externally-managed-environment on heisenbug
+    # We therefore need to create an environment, e.g. with:
+    python3 -m venv /import/exception-dump/ulrich/python-venv
+    export PATH=/import/exception-dump/ulrich/python-venv/bin:$PATH
```
Comment (vikaskurapati) [inline Documentation/heisenbug.rst:37]: Is this path accessible to everyone when they run it?

File: Documentation/heisenbug.rst [hunk 2]
```diff
@@ -10,67 +10,84 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
...
+    export XDMFWRITER_BLOCK_SIZE=4096
+    export ASYNC_BUFFER_ALIGNMENT=4096
+    source /etc/profile.d/modules.sh
+    export SEISSOL_ASAGI_MPI_MODE=OFF
+    ulimit -Ss 2097152
+    export OMP_NUM_THREADS=10
     export OMP_PLACES="cores"
     export OMP_PROC_BIND=spread
-    ./launch ./SeisSol_RelWithDebInfo_ssm_86_cuda_4_elastic ./parameters.par
+    seissol-launch SeisSol_Release_ssm_86_cuda_4_elastic ./parameters.par
```
Comment (vikaskurapati) [inline Documentation/heisenbug.rst:69]: Can we have the creation of `seissol-launch` here in the bash script like we did for the LUMI/frontera bash script?

File: Documentation/heisenbug.rst [hunk 3]
```diff
@@ -10,67 +10,84 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
...
+    export SEISSOL_ASAGI_MPI_MODE=OFF
+    ulimit -Ss 2097152
     # Note that it is possible to increase OMP_NUM_THREADS 
     # This will speed up (the rare) portions of the code running only CPUs, e.g. the wiggle factor calculation 
-    export OMP_NUM_THREADS=1 
+    export OMP_NUM_THREADS=1
     export OMP_PLACES="cores"
     export OMP_PROC_BIND=spread
-    mpirun -n 2 --map-by ppr:1:numa:pe=2 --report-bindings ./launch ./SeisSol_RelWithDebInfo_ssm_86_cuda_4_elastic ./parameters.par
+    mpirun -n 2 --map-by ppr:2:numa:pe=$OMP_NUM_THREADS --report-bindings seissol-launch SeisSol_Release_ssm_86_cuda_4_elastic ./parameters.par
```
Comment (vikaskurapati) [inline Documentation/heisenbug.rst:93]: The same as before for the `seissol-launch`

File: Documentation/heisenbug.rst (hunk 2 above)
Comment (davschneller) [reply Documentation/heisenbug.rst:69]: Actually, a `seissol-launch` script is already being generated on each build—in fact, it should also work for LUMI with slurm already now (we just don't use it in the docs there yet :) ).

File: Documentation/heisenbug.rst (hunk 1 above)
Comment (Thomas-Ulrich) [reply Documentation/heisenbug.rst:37]: no, it is just an example, there the "e.g."

**Pull request merged**